packages/ui-kit-components/src/components/IndexTickers/IndexTickers.test.tsx

```diff
@@ -0,0 +1,91 @@
+import '@testing-library/jest-dom';
+
+import { render, screen } from '@testing-library/react';
+
+import IndexTickers from './IndexTickers';
+
+describe('Given render the index tickers component', () => {
```

Comment:
Naming convention: the name of the entire test suite should only be the component name `IndexTickers Component`